packages/shared/src/components/post/PostComments.tsx

```diff
@@ -92,7 +92,7 @@ export function PostComments({
         requestMethod(
           `${apiUrl}/graphql`,
           POST_COMMENTS_QUERY,
-          { postId: id },
+          { postId: id, initial: comments },
```

Comment:
@rebelchris I meant for this case as well. So we can always be sure that the keys are matched. Probably just a personal preference, but we can let this go as it is not quite critical. Just for typos.
```suggestion
          { postId: id, [initialDataKey]: comments },
```